### PR TITLE
add unittests with snapshots

### PR DESCRIPTION
**Pull request by matthyx**
to be run with
```
helm unittest -u charts/kubescape-cloud-operator/
```

### CODE REVIEW

**Comment by dwertent** [comment]
I think we should add this to the github actions so it would run when opening a PR

**Comment by matthyx** [comment]
> I think we should add this to the github actions so it would run when opening a PR

yes good point, for the moment I am using it to ensure I don't mess things up with the refactoring...
Instructions for docker run: https://github.com/helm-unittest/helm-unittest#docker-usage

File: charts/kubescape-cloud-operator/tests/snapshot_test.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+suite: Full Snapshot
+tests:
+  - it: matches the snapshot
+    asserts:
+      - matchSnapshot: {}
+    capabilities:
+      apiVersions:
+        - batch/v1/CronJob
+    set:
+      account: 9e6c0c2c-6bd0-4919-815b-55030de7c9a0
```
Comment (dwertent) [inline charts/kubescape-cloud-operator/tests/snapshot_test.yaml:10]: I hope this is not a real account :)

Comment (matthyx) [reply charts/kubescape-cloud-operator/tests/snapshot_test.yaml:10]: it was our 1-4-3 account, I don't think it's critical?

**Pull request merged**